### PR TITLE
Skip undeserializable history events

### PR DESCRIPTION
**Pull request by nebolax**

### CODE REVIEW

File: rotkehlchen/db/history_events.py [hunk 1]
```diff
@@ -186,6 +186,7 @@ def get_history_events(
                 deserialized = HistoryBaseEntry.deserialize_from_db(entry)
             except (DeserializationError, UnknownAsset) as e:
                 log.debug(f'Failed to deserialize history event {entry} due to {str(e)}')
+                continue
```
Comment (LefterisJP) [inline rotkehlchen/db/history_events.py:189]: ouch

File: rotkehlchen/tests/api/test_ethereum_transactions.py [hunk 2]
```diff
@@ -932,6 +944,22 @@ def query_transactions(rotki) -> None:
         gevent.joinall(rotki.greenlet_manager.greenlets)
         rotki.task_manager._maybe_decode_evm_transactions()
         gevent.joinall(rotki.greenlet_manager.greenlets)
+        with dbevents.db.user_write() as write_cursor:
+            # Add an event with random values. Important thing is that asset doesn't exist.
+            # We do it to check that this event will be skipped during deserialization
```
Comment (LefterisJP) [inline rotkehlchen/tests/api/test_ethereum_transactions.py:949]: Is it to skip during deserialization or to not fail when there is an even with unknown asset?

Comment (nebolax) [reply rotkehlchen/tests/api/test_ethereum_transactions.py:949]: Eh... both? This event won't be returned

**Review by prettyirrelevant — APPROVED** [review]
LGTM

**Review by LefterisJP — APPROVED** [review]
lgtm

**Pull request merged**